readme.md

```diff
@@ -90,6 +90,11 @@ ___Note!___ Project is still under heavy development
 
 ## Changelog
 
+## v0.3.0
+* $responsiveClassScheme added to _variables.scss for freedom to specify the syntax of responsive classes. The default syntax is r-{breakpoint}-{selector}.
+* Responsive classes are created with createResponsiveClasses mixin.
+* .clearfix class has been deprecated
+
```

Comment:
+ Couple of mixins were renamed / parameters changed?
+ Adds responsive styles for layout classes